internal/constants/database.go

```diff
@@ -5,4 +5,8 @@ const (
 	ImageCollection      = "mined_images"
 	BatchCollection      = "batches"
 	BatchImageCollection = "batch_images"
+	BatchCollectionMe = "batchCollection"
+	BatchMinedCollection = "batchimageCollection"
+
+	// please lets leave the database like this till after testing. I uploaded some dummy data on these
 )
```

Comment:
this will cause issues on the frontend though.
so the images in the batches they created will not be shown
you get me.

Reply:
after testing I will remove it. I just to do that for testing purpose. Cos I am not sure everything will be ready for testing soon. and Mark is requesting for testing. will remove it

Reply:
ok